src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/OffenderService.kt

```diff
@@ -97,4 +98,51 @@ OffenderService(
       }
     }
   }
+
+  @Transactional
+  fun updateOffenderDetails(nomsId: String, request: UpdateOffenderDetailsRequest) {
+    val licenceStatuses = listOf(
```

Comment:
maybe the variable name could reflect what this list represents? I think it might mean "not inactive" licences?

In the past I've been on projects where we would centralise common lists of statuses as static fields in the Enum, potentially extending the domain if suitable `LicenceStatus.ALL_ACTIVE` `PolicyStatus.ALL_ON_TRIAL`  etc.. not sure we've hit that limit yet